lib/browser/otter-browser.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: 
+
+module Browser
+  class OtterBrowser < Base
```

Comment:
Similar code found in 8 other locations (mass = 21) ([lib/browser/otter-browser.rb#4](https://github.com/fnando/browser/blob/9432836727430d1ea738f4ef36f565eb4225524c/lib/browser/otter-browser.rb#L4-L18), [lib/browser/alipay.rb#4](https://github.com/fnando/browser/blob/9432836727430d1ea738f4ef36f565eb4225524c/lib/browser/alipay.rb#L4-L18), [lib/browser/electron.rb#4](https://github.com/fnando/browser/blob/9432836727430d1ea738f4ef36f565eb4225524c/lib/browser/electron.rb#L4-L19), [lib/browser/firefox.rb#4](https://github.com/fnando/browser/blob/9432836727430d1ea738f4ef36f565eb4225524c/lib/browser/firefox.rb#L4-L18), [lib/browser/micro\_messenger.rb#4](https://github.com/fnando/browser/blob/9432836727430d1ea738f4ef36f565eb4225524c/lib/browser/micro_messenger.rb#L4-L18), [lib/browser/nokia.rb#4](https://github.com/fnando/browser/blob/9432836727430d1ea738f4ef36f565eb4225524c/lib/browser/nokia.rb#L4-L18), [lib/browser/phantom\_js.rb#4](https://github.com/fnando/browser/blob/9432836727430d1ea738f4ef36f565eb4225524c/lib/browser/phantom_js.rb#L4-L18), [lib/browser/uc\_browser.rb#4](https://github.com/fnando/browser/blob/9432836727430d1ea738f4ef36f565eb4225524c/lib/browser/uc_browser.rb#L4-L18), and [lib/browser/weibo.rb#4](https://github.com/fnando/browser/blob/9432836727430d1ea738f4ef36f565eb4225524c/lib/browser/weibo.rb#L4-L18))